python/paddle/distributed/parallel.py

```diff
@@ -140,21 +140,6 @@ def _check_var_exists(var_name):
         http_server.start()
     wait_server_ready([ParallelEnv().trainer_endpoints[0]])
```

Comment:
This line can be moved to line #169 (just after the nccl initialization) to decrease the wait time.